googleapiclient/discovery_cache/__init__.py

```diff
@@ -19,6 +19,9 @@
 import logging
 import datetime
 
+
+logger = logging.getLogger(__name__)
```

Comment:
I was remaining consistent with the style [here](https://github.com/google/google-api-python-client/blob/30125120b48860d00ba4e96d56a86d9d02a90960/googleapiclient/discovery_cache/file_cache.py#L37), but I can make this change if this repo prefers the all-caps style.